hathorlib/base_transaction.py

```diff
@@ -566,6 +587,13 @@ def to_json(self, *, decode_script: bool = False) -> Dict[str, Any]:
             data['decoded'] = self.to_human_readable()
         return data
 
+    def is_standard_script(self) -> bool:
+        """Return True if this output has a standard script."""
+        from hathorlib.scripts import P2PKH, MultiSig, parse_address_script
+        parsed_output = parse_address_script(self.script)
+        standard_types = (P2PKH, MultiSig)
```

Comment:
We should move this tuple to the `TxInput` (instead of keeping it inside the method).

Reply:
Or at least move it to something like `hathorlib.scripts.STANDARD_TYPES`, since it's a constant. At least being the scripts module would make it much harder to miss when we add a new standard type.

Reply:
I agree with you. I think I will move to `TxOutput`